hw06/src/main/java/org/otus/hw06/storage/StorageImpl.java

```diff
@@ -0,0 +1,52 @@
+package org.otus.hw06.storage;
+
+import org.otus.hw06.banknote.Banknote;
+import org.otus.hw06.banknote.Nominal;
+
+import java.util.*;
+
+public class StorageImpl implements Storage {
+    private Map<Nominal, List<Banknote>> listMap;
```

Comment:
```suggestion
    private List<Cell> cells;
```

Можно ввести явный объект Ячейка (Cell). 
```java
class Cell {
  private Nominal nominal;
  private List<Banknote> banknotes;
}
```

В этом ДЗ кстати можно не учитывать каждую банкноту, а учитывать только их количество (тогда Banknote не нужен):
```java
class Cell {
  private Nominal nominal;
  private int count;
}
```